tests/utest/test_utils.py

```diff
@@ -190,3 +191,68 @@ def test_invalid_pattern_type(self):
         with pytest.raises(ValueError) as error:
             pattern_type(r"[\911]")
         assert r"Invalid regex pattern: bad escape \\9 at position 1" in str(error)
+
+
+@pytest.mark.parametrize(
+    "robot_version, rule_version, should_pass",
+    [
+        ("6.0.2", ">=6", True),
+        ("6.1a2.dev1", ">=6", True),
+        ("6.0rc2.dev1", ">=6", False),
+        ("6.0pre2.dev1", ">=6", False),
```

Comment:
Shouldn't these be `True`?

Reply:
On first look I also thought that, but its actually realase candidate of the 6.0 version (so its still not 6 oficially). We dont want to enable/disable rules for version that is yet to be oficially released - after all Robot Framework for example automatically switch to dev version right after the release

Reply:
It's good to better understand how version specifiers work. Thanks for clarifying.